packages/xchain-thorchain/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xchainjs/xchain-thorchain",
-  "version": "0.23.0",
+  "version": "0.24.0",
```

Comment:
Suggestions: Let's start with `xchain-thorchain@0.24.0-alpha.1` - so we can add fixes / tweaks easily by increasing `alpha` without bumping the final version number.